week1/homework/assignments.md

```diff
@@ -2,15 +2,19 @@
 
 **Assignment 1:**
 
-What is an ARN: `?`
+What is an ARN: `Amazon Resource Names (ARNs) uniquely identify AWS resources. We require an ARN when you need to specify a resource unambiguously across all of AWS, such as in IAM policies, Amazon Relational Database Service (Amazon RDS) tags, and API calls.`
 
 **Assignment 2:**
 
-Response from the command: `?`
+Response from the command: `'us-east-1' is a supported AWS Region`
 
 **Assignment 3:**
 
-Command to upload file with the AWS CLI: `?`
+Command to upload file with the AWS CLI:
```

Comment:
You need to find the command that you can copy paste into a terminal to upload it. Ideally with some docs. What you have here is a couple of strings.